### PR TITLE
Improve wording

### PR DESCRIPTION
**Pull request by XmM14**

### CODE REVIEW

**Comment by XmM14** [comment]
I may input the wrong action. As a new contributor take my apology first!

File: files/en-us/learn/common_questions/how_does_the_internet_work/index.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ So we are connected to the telephone infrastructure. The next step is to send th
 
 If you want to send a message to a computer, you have to specify which one. Thus any computer linked to a network has a unique address that identifies it, called an "IP address" (where IP stands for _Internet Protocol_). It's an address made of a series of four numbers separated by dots, for example: `192.168.2.10`.
 
-That's perfectly fine for computers, but we human beings have a hard time remembering that sort of address. To make things easier, we can alias an IP address with a human-readable name called a _domain name_. For example (at the time of writing; IP addresses can change) `google.com` is the domain name used on top of the IP address `142.250.190.78`. So using the domain name is the easiest way for us to reach a computer over the Internet.
+That's perfectly fine for computers, but we human beings have a hard time remembering that sort of address. To make things easier, we can alias an IP address with a human-readable name called a _domain name_. e.g., `google.com` is the domain name used on top of the IP address `142.250.190.78`(at the time of writing; IP addresses may change). So using the domain name is the easiest way for us to reach a computer over the Internet.
```
Comment (Josh-Cena) [inline files/en-us/learn/common_questions/how_does_the_internet_work/index.md:89]:
```suggestion
That's perfectly fine for computers, but we human beings have a hard time remembering that sort of address. To make things easier, we can alias an IP address with a human-readable name called a _domain name_. For example, `google.com` is the domain name used on top of the IP address `142.250.190.78` (at the time of writing; IP addresses may change). So using the domain name is the easiest way for us to reach a computer over the Internet.
```

Comment (XmM14) [reply files/en-us/learn/common_questions/how_does_the_internet_work/index.md:89]: It looks fine!

**Review by estelle — APPROVED** [review]
thank you

**Review by dipikabh — APPROVED** [review]

**Comment by dipikabh** [comment]
Hello @XmM14, we'd appreciate if you can resolve the conflicts in the file for this change to be merged. Thanks!

**Pull request closed without merging**